src/style.css

```diff
@@ -0,0 +1,91 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: bold;
+  font-weight: 900;
+}
+
+.container {
+  margin-top: 20px;
+  position: relative;
+}
+
+.image {
+  position: absolute;
+}
+
+.image-small {
+  top: 15px;
+  left: 13px;
+  height: 11px;
+  width: 11px;
+}
+
+.image-big {
+  top: 25px;
+  left: 26px;
+  height: 19px;
+  width: 19px;
+}
+
+.search {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  font-weight: inherit;
+  width: 100%;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+}
+
+.search-big {
+  padding-left: 62px;
+  background-size: 19px;
+  background-position: 26px;
+  height: 70px;
+  font-size: inherit;
+}
+
+.search-small {
+  padding-left: 33px;
+  background-size: 11px;
+  background-position: 13px;
+  font-size: 14px;
+  height: 42px;
+}
```

Comment:
There should be default `search` class(big or small, you decide), and modified one `search--small` or `search--big` with only styles that differs from default `search` class

Reply:
done